composer.json

```diff
@@ -22,11 +22,11 @@
     }
   },
   "require": {
-    "php": ">=7.3",
+    "php": "^8.0",
     "nikic/php-parser": "^4.0"
```

Comment:
For PHP>=8.0, this dependency should be bumped to at least ^4.10.5. Because only 4.10.0 supports PHP8 attributes

Reply:
https://github.com/goaop/parser-reflection/pull/122/commits/e96f9bbaa0be7f40cda786257aa46d526558f6c4